source/pug/partials/hero-unit.pug

```diff
@@ -1,5 +1,5 @@
-header
-  .header-content.container
+header.page-header
```

Comment:
Maybe we move the `header {}` selector into `.nsf-page header`? That way other pages can define their own header, and we also don't need to add another classname?